modules/templates/util_render.go

```diff
@@ -114,6 +114,13 @@ func (ut *RenderUtils) RenderIssueTitle(text string, metas map[string]string) te
 	return template.HTML(renderedText)
 }
 
+// RenderIssueSimpleTitle only renders with emoji and inline code block
+func (ut *RenderUtils) RenderIssueSimpleTitle(text string) template.HTML {
+	ret := ut.RenderEmoji(text)
+	ret = renderCodeBlock(ret)
+	return ret
+}
```

Comment:
This could almost be used to render commit titles as well. I guess the only difference is that commit titles also render issue links.

Reply:
No, commit subject has its own "render"